data/src/server.rs

```diff
@@ -77,6 +77,7 @@ impl Map {
                         *password = Some(pass);
                     },
                     Sasl::Plain { password: Some(_), password_file: None, .. } => {},
+                    Sasl::External { .. } => {},
```

Comment:
I think we can turn the entire `match sasl` into a `if let`:

```rust
if let Sasl::Plain { password: password @ None, password_file: Some(pass_file), .. } = sasl {
    let pass = fs::read_to_string(pass_file)?;
    *password = Some(pass);
}
```

Reply:
resolved by f47da01